.github/workflows/ci.yml

```diff
@@ -4,41 +4,95 @@ on: [push, pull_request]
 
 jobs:
   build:
-    name: ${{ matrix.os }} ${{ matrix.python }} ${{ matrix.name }}
+
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: true
       matrix:
-        # os: [ubuntu-latest, macos-latest, windows-latest]
-        os: [ubuntu-latest]
-        python: ['3.6', '3.7', '3.8', '3.9']
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        python-version: [ '3.6', '3.7', '3.8', '3.9' ]
+
+        # FIXME: Unfortunately, building for Python 3.9 on Windows fails with "fatal error LNK1181: cannot open input file 'hdf5.lib'"
+        # https://github.community/t/how-to-conditionally-include-exclude-items-in-matrix-eg-based-on-branch/16853/6
+        exclude:
+          - os: windows-latest
+            python-version: '3.9'
+    env:
+      OS: ${{ matrix.os }}
+      PYTHON: ${{ matrix.python-version }}
+
+    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python }}
-      uses: actions/setup-python@v1
+
+    - name: Acquire sources
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
       with:
-        python-version: ${{ matrix.python }}
-    - name: Install APT packages
-      if: contains(${{ matrix.os }}, 'ubuntu')
-      run: |
-        sudo apt install libbz2-dev libhdf5-serial-dev liblzo2-dev
-    - name: Install dependencies
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Debian packages
+      if: contains(matrix.os, 'ubuntu')
```

Comment:
use rather 
```
if: runner.os == 'Linux'
```
than checking system in the matrix.

Reply:
Thanks for this suggestion!